plugins/out_stackdriver/stackdriver.h

```diff
@@ -104,12 +104,25 @@ struct flb_stackdriver {
     flb_sds_t pod_name;
     flb_sds_t container_name;
     flb_sds_t node_name;
-    bool k8s_resource_type;
+    bool is_k8s_resource_type;
 
     flb_sds_t labels_key;
     flb_sds_t local_resource_id;
     flb_sds_t tag_prefix;
 
+    /* generic resources */
+    flb_sds_t location;
+    flb_sds_t namespace_id;
+    bool is_generic_resource_type;
+
+    /* generic_node specific */
+    flb_sds_t node_id;
+
+    /* generic_task specific */
+    flb_sds_t job;
+    flb_sds_t task_id;
```

Comment:
[Optional] Does it make sense to create a sub-struct for the generic resource stuff?

Reply:
In the future, yes. (ie. when I abstract the resources out) but for now following the current structure for consistency makes more sense.